tinker/admin/blink_roles/__init__.py

```diff
@@ -11,8 +11,7 @@ class BlinkRolesView(FlaskView):
     route_base = '/admin/blink-roles'
 
     def before_request(self, args):
-        if 'Administrators' not in session['groups']:
-            abort(403)
+        admin_permissions('route_base', self, args)
```

Comment:
should the 'route_base' string be the variable route_base?

(there are a bunch of these throughout the files